Numsense/Russian.fs

```diff
@@ -0,0 +1,154 @@
+﻿module internal Ploeh.Numsense.Russian
+
+open Ploeh.Numsense.InternalDsl
+
+type internal Gender =
+    | Masculine
+    | Feminine
+    | Neuter
+
+let rec internal toRussianImp gender x =
+
+    let simplify gender prefix factor x =
+        let remainder = x % factor
+        if remainder = 0
+        then prefix
+        else sprintf "%s %s" prefix (toRussianImp gender (remainder))
+
+    let impUnderThousand gender x =
+        let simplify = simplify gender
+        match x with

```

Comment:
The compiler doesn't understand that, though, so now there's a compiler warning in the build, and I take a dim view on those...